src/input-nested-select/input-nested-select.tsx

```diff
@@ -135,7 +137,7 @@ export const InputNestedSelect = <V1, V2, V3>({
     const getDisplayValue = (): string => {
         const { label, value } = selectedItem;
 
-        if (valueToStringFunction) {
+        if (valueToStringFunction && value) {
```

Comment:
can you elaborate on why this check is needed? I thought the `isEmpty` check before this ensures there is a selection

Reply:
my bad. just now type yelling at me.  now is good.
i will remove the value.